api/v1beta1/verifier_types.go

```diff
@@ -23,13 +23,16 @@ import (
 
 // VerifierSpec defines the desired state of Verifier
 type VerifierSpec struct {
-	// Important: Run "make" to regenerate code after modifying this file
+	// Important: Run "make install-crds" to regenerate code after modifying this file
 
 	// Name of the verifier
-	Name string `json:"name,omitempty"`
+	Name string `json:"name"`
+
+	// Version of the verifier plugin
+	Version string `json:"version,omitempty"`
 
 	// The type of artifact this verifier handles
-	ArtifactTypes string `json:"artifactTypes,omitempty"`
+	ArtifactTypes string `json:"artifactTypes"`
```

Comment:
curious, why the change here?

Reply:
as discussed, ArtifactType will be required going forward